adam-cli/src/main/scala/org/bdgenomics/adam/cli/CompareADAM.scala

```diff
@@ -173,13 +173,19 @@ class CompareADAM(protected val args: CompareADAMArgs) extends ADAMSparkCommand[
 
   def run(sc: SparkContext, job: Job): Unit = {
 
-    if (args.listComparisons) {
+    // Work around for -list_comparisons (doesn't require @Arguments)
+    if (!args.listComparisons && (args.input1Path == null || args.input2Path == null)) {
+      val options = new CompareADAMArgs
+      val parser = new CmdLineParser(options)
+      println("\n2 Arguments (files) must be given as input.\n")
+      parser.printUsage(System.err)
+      sys.exit(1)
+    } else {
       println("\nAvailable comparisons:")
       DefaultComparisons.comparisons.foreach {
         generator =>
           println("\t%10s : %s".format(generator.name, generator.description))
       }
-
       return
```

Comment:
Commenting this return statement out produces an error. What exactly does it do?